### PR TITLE
Sonatype Deployment

### PR DESCRIPTION
**Pull request by blakesteve**
Resolves the issue where snapshots were not being deployed to Sonatype

### CODE REVIEW

**Comment by germanattanasio** [comment]
2 comments:
 * Can we move the `cd` and the python script to the `.utility` folder?

File: core/build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ apply plugin: 'checkstyle'
 apply plugin: 'eclipse'
 
 version = '3.5.4-SNAPSHOT'
```
Comment (germanattanasio) [inline core/build.gradle]: This version was not updated to be 3.5.5.

**Comment by germanattanasio** [comment]
@schen22 can we leave the version as 3.5.4 for now?

**Comment by schen22** [comment]
@germanattanasio - changed back to 3.5.4

**Pull request merged**